extract_endpoint/tests/test_azure_utils.py

```diff
@@ -41,6 +42,7 @@ def check_file_in_azure(azure_service: blob.BlockBlobService,
     assert actual_blob.content == contents
 
 
+@pytest.mark.usefixtures('azure_service')
 def test_upload_bytes(azure_emulator_coords: azure_utils.StorageCoordinates,
                       azure_service: blob.BlockBlobService,
```

Comment:
azure_service already in parameters

Reply:
Good catch! Unfortunately, without adding the mark.usefixtures, the skip doesn't work :(